src/components/organisms/NewResourceWizard/NewResourceWizard.tsx

```diff
@@ -91,6 +129,32 @@ const NewResourceWizard = () => {
     [registeredKindHandlers, resourceMap]
   );
 
+  const generateExportFileName = async () => {
+    if (fileMap[ROOT_FILE_ENTRY] && selectedFolder.startsWith(fileMap[ROOT_FILE_ENTRY].filePath)) {
+      const currentFolder = selectedFolder.split(`${fileMap[ROOT_FILE_ENTRY].filePath}`).pop();
+
+      if (currentFolder) {
+        setSelectedFolder(currentFolder.slice(1));
+      } else {
+        setSelectedFolder(ROOT_FILE_ENTRY);
+      }
+      return;
+    }
+
+    const resourceNames = Object.keys(resourceMap).map(id => resourceMap[id].name);
```

Comment:
Could use `Object.values` directly here, but let's also filter them by `filePath` that starts with the `selectedFolder`